content/pt-br/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,58 @@
+
+<!-- body -->
+
+Um _LimitRange_ fornece restrições que podem:
+
+- Aplicar o uso mínimo e máximo de recursos computacionais por pod ou contêiner em um _namespace_.
+- Impor a solicitação de armazenamento mínimo e máximo por _PersistentVolumeClaim_ em um _namespace_.
+- Impor a proporção entre solicitação e limite para um recurso em um _namespace_.
+- Definir a solicitação/limite padrão para recursos computacionais em um _namespace_ e utilizá-los automaticamente nos contêineres em tempo de execução.
+
+## Ativando o LimitRange
+
+O suporte a _LimitRange_ foi ativado por padrão desde o Kubernetes 1.10.
+
+Um _LimitRange_ é aplicado em um _namespace_ específico quando há um objeto _LimitRange_ nesse _namespace_.
+
+O nome de um objeto _LimitRange_ deve ser um [nome de subdomínio DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) válido.
+
+### Visão geral do Limit Range
+
+- O administrador cria um _LimitRange_ em um _namespace_.
+- Os usuários criam recursos como pods, contêineres e _PersistentVolumeClaims_ no _namespace_.
+- O controlador de admissão `LimitRanger` impõe padrões e limites para todos os pods e contêineres que não definem os requisitos de recursos computacionais e rastreia o uso para garantir que não exceda o mínimo, o máximo e a proporção de recursos definidos em qualquer _LimitRange_ presente no _namespace_.
+- Se estiver criando ou atualizando um recurso (Pod, Container, _PersistentVolumeClaim_) que viola uma restrição LimitRange, a solicitação ao servidor da API falhará com um código de status HTTP `403 FORBIDDEN` e uma mensagem explicando a restrição violada.
+- Se um _LimitRange_ for ativado em um _namespace_ para recursos computacionais como `cpu` e `memória`, os usuários deverão especificar solicitações ou limites para esses valores. Caso contrário, o sistema pode rejeitar a criação do pod.
+- As validações de _LimitRange_ ocorrem apenas no estágio de Admissão de Pod, não em Pods em Execução.
+
+Alguns exemplos de politicas que podem ser criadas utilizando os intervalos de limite são:
+
+- Em um cluster de 2 nós com capacidade de 8 GiB de RAM e 16 núcleos, restrinja os Pods em um namespace para solicitar 100m de CPU com um limite máximo de 500m para CPU e solicitar 200Mi para memória com um limite máximo de 600Mi para memória.
+- Defina o limite e a solicitação de CPU padrão para 150m e a solicitação padrão de memória para 300Mi para contêineres iniciados sem solicitações de CPU e memória em suas especificações.
+
+Caso os limites totais do namespace sejam menores que a soma dos limites dos Pods/Contêineres, pode haver contenção por recursos. Nesse caso, os contêiners ou Pods não serão criados.
```

Comment:
Hi, in line 40, was used "contêineres", but in line 42 was written "contêiners".
Keep attention and look for in the article if exist any word with this typo.
May you fix that, please?

Reply:
I just changed, thanks for the comment!